gcp/modules/node_pools/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "cluster_name" {
+  type = string
+}
+
+variable "cluster_node_pool_count" {
+  type = number
+}
+
+variable "cluster_node_type" {
+  type = string
+}
+
+variable "cluster_node_disk_size" {}
+
+variable "region" {}
```

Comment:
Same as for the cluster, are there any sane defaults you think we could/should apply for these variables?

Reply:
I think I can add some defaults here.